sentry_sdk/consts.py

```diff
@@ -75,6 +75,8 @@ def __init__(
         traces_sample_rate=None,  # type: Optional[float]
         traces_sampler=None,  # type: Optional[TracesSampler]
         auto_enabling_integrations=True,  # type: bool
+        auto_session_tracking=True,  # type: bool
+        session_mode="application",  # type: str
```

Comment:
not sure about this default, I think request-mode is what we currently instrument for? not sure how this works cc @mitsuhiko

Reply:
https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/client.py#L108